vendor/bat-native-ledger/src/bat/ledger/internal/legacy/bat_state.cc

```diff
@@ -45,7 +45,8 @@ void LegacyBatState::OnLoad(
   ledger::ClientProperties state;
   const ledger::ClientState client_state;
   if (!client_state.FromJson(data, &state)) {
-    BLOG(0, "Failed to load client state: " << data);
+    BLOG(0, "Failed to load client state");
+    BLOG(1, "Client state contents: " << data);
```

Comment:
Elsewhere in ads and ledger we have used `6` for responses, would be good to use the same value so that these are only shown when the verbosity is 6 or higher.